src/acp/rule.ts

```diff
@@ -315,29 +347,74 @@ export function getRule(
   return foundThing;
 }
 
+/**
+ * ```{note} There is no Access Control Policies specification yet. As such, this
+ * function is still experimental and subject to change, even in a non-major release.
+ * ```
+ *
+ * Get the [[ResourceRule]] with the given name from an Resource's Access Control
+ * Resource.
+ *
+ * @param resourceWithAcr The Resource whose Access Control Resource contains the given [[ResourceRule]].
+ * @param name Name that identifies this [[ResourceRule]].
+ * @returns The requested [[ResourceRule]], if it exists, or `null` if it does not.
+ */
+export function getResourceRule(
+  resourceWithAcr: WithAccessibleAcr,
+  name: string
+): ResourceRule | null {
+  const acr = internal_getAcr(resourceWithAcr);
+  const acrUrl = getSourceUrl(acr);
+  const url = new URL(acrUrl);
+  url.hash = `#${name}`;
+  const foundThing = getThing(acr, url.href);
+  if (foundThing === null || getUrl(foundThing, rdf.type) !== acp.Rule) {
```

Comment:
```suggestion
  if (foundThing === null || !isRule(foundThing) {
```
Would this work ? I find it slightly easier to read.

Reply:
Good point, applied for Policies too: c771f5ebff3fb468eae61b4baad9c2096d400a3c and ead7e849be7348eda4a9152c4e1bb3ec2b1f9a18.